### DISCUSSION TITLE
Changes not getting reflected on localhost web after the code changes

### DISCUSSION THREAD

**Discussion by nikita-sinha-411** [apache/superset#24034] [Q&A / Help]
When I am trying to make some code changes after creating my own fork for superset, I am not able to see those changes on localhost. i am running the python and react code locally.

**Comment by eschutho** [comment]
Hi, are you running the webpack dev server with `npm run dev-server`? That will give you live updates. You can also run `npm run dev` and you will see the updates after refreshing.

**Reply by nikita-sinha-411** [reply]
I am running the webpack dev server with npm run dev-server. It isn't working that way. Please guide further

**Comment by fikryset** [comment]
Hello, any tips for this problem? This is happening on my local too, need to rerun the `npm run dev-server` to see the changes, refresh doesn't update the webpage. I am running superset with python flask too

**Comment by mayur249** [comment]
Hey @nikita-sinha-411, did you solved this issue, I am also facing the same thing. I am making changes in my local for FE but I am not able to see my changes in the browser
